.github/stale.yml

```
@@ -0,0 +1,16 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 365
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 90
+# Limit to only `issues` or `pulls`
+only: issues
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - "Type: Feature"
+  - "Type: Understood"
+# Label to use when marking an issue as stale
+staleLabel: "Status: Stale"
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as "stale" because it has not had
+  any activity for a while. It will be closed in 90 days if no further activity occurs.
```

Last thing - let's remain `Thank you for your contributions.` part, you can just apply next suggestion

```suggestion
markComment: >
  This issue has been automatically marked as "stale" because it has not had
  any activity for a while. It will be closed in 90 days if no further activity occurs. 
  Thank you for your contributions.
```